src/linux/Packaging.Linux/pack.sh

```diff
@@ -100,6 +100,26 @@ INSTALL_TO="$DEBROOT/usr/local/share/gcm-core/"
 LINK_TO="$DEBROOT/usr/local/bin/"
 mkdir -p "$DEBROOT/DEBIAN" "$INSTALL_TO" "$LINK_TO" || exit 1
 
+# Fall back to host architecture if no explicit runtime is given.
+if test -z "$RUNTIME"; then
+    HOST_ARCH="`dpkg-architecture -q DEB_HOST_ARCH`"
+
+    case $HOST_ARCH in
+        amd64)
+            RUNTIME="linux-x64"
+            ;;
+        arm64)
+            RUNTIME="linux-arm64"
+            ;;
+        armhf)
+            RUNTIME="linux-arm"
+            ;;
+        *)
+            die "Could not determine host architecture!"
+            ;;
+    esac
+fi
+
```

Comment:
If we move back to requiring `$RUNTIME` and use the `uname -m` command instead in `build.sh` we won't need this.

Reply:
Refer to https://github.com/git-ecosystem/git-credential-manager/pull/1757#discussion_r1831288265